### DISCUSSION TITLE
我跟个傻子一样，不会部署，谁会的教我一下可以吗

### DISCUSSION THREAD

**Discussion by xueqi5201314** [LagrangeDev/Lagrange.Core#602] [Q&A] [answered]
如题

**Comment by sisi0318** [comment]
cool

**Comment by Decrabbityyy** [comment]
you can ask ai ,like chatgpt :)

**Comment by jiangyin14** [comment]
看文档去 bing能搜到

**Comment by shenxgan** [comment]
不是有docker版本吗？可以试试

**Comment by RTAkland** [comment]
。。。。下载可执行文件,chmod,./Lagrange.OneBot。三步完成（（（（（

**Comment by NoirHare** [comment, marked as answer]
在 release(更新较慢) 或 action(更新较快) 下载对于平台的二进制，如果是 release 的，直接运行并根据提示修改配置，如果是 action 的，需要安装对于版本的 .Net 运行时，然后运行并根据提示修改配置
